scripts/technique-files

```diff
@@ -96,6 +101,12 @@ find_directories() {
       fi
 }
 
+# find extra 'special' files in initial-promises root (promises.cf, failsafe.cf, ...)
+find_initial() {
+  if [ ${INCLUDE_INITIAL} -eq 1 ] && [ -d ${directory}/initial-promises/node-server ]; then
+    find "${directory}/initial-promises/node-server/" -maxdepth 1 -not -wholename "*/.git/*" -type f
```

Comment:
since directory can be any directory, including initial-promises itself, I'd prefer something like
node_server=$(find "${directory}" -type d -name node-server)
and use this variable instead of the literal ${directory}/initial-promises/node-server